tests/FunctionalTest.php

```diff
@@ -45,6 +45,12 @@ public function testBuildCssIfUsed(): void
 
         $asset = $assetMapper->getAsset('app.css');
         $this->assertInstanceOf(MappedAsset::class, $asset);
-        $this->assertStringContainsString('color: red', $asset->content);
+
+        if (null === $asset->content) {
```

Comment:
Hm, I think we should base this check on `Kernel::VERSION_ID` instead to be clearer and more robust?

Reply:
Used it first, but one can have different versions of AssetMapper and Kernel :/

Reply:
Ah, I see... then the current solution is OK I think